native/src/components/Consent.tsx

```diff
@@ -0,0 +1,59 @@
+import { useFocusEffect } from '@react-navigation/native'
+import React, { ReactElement, useCallback, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { View, StyleSheet } from 'react-native'
+import styled from 'styled-components/native'
+
+import { capitalizeFirstLetter } from 'api-client'
+
+import buildConfig from '../constants/buildConfig'
+import appSettings, { ExternalSourcePermission } from '../utils/AppSettings'
+import { updateSourcePermissions } from '../utils/helpers'
+import { reportError } from '../utils/sentry'
+import Caption from './Caption'
+import ConsentSection from './ConsentSection'
+import Layout from './Layout'
+
+export const ItemSeparator = styled.View`
+  background-color: ${props => props.theme.colors.textDecorationColor};
+  height: ${StyleSheet.hairlineWidth}px;
+`
+
+const Consent = (): ReactElement | null => {
+  const [externalSourcePermissions, setExternalSourcePermissions] = useState<ExternalSourcePermission[]>([])
+  const { t } = useTranslation('consent')
+
+  useFocusEffect(
+    useCallback(() => {
+      appSettings.loadExternalSourcePermissions().then(setExternalSourcePermissions).catch(reportError)
+    }, []),
+  )
+
+  const onPress = (type: string, allowed: boolean) => {
+    const updatedSources = updateSourcePermissions(externalSourcePermissions, { type, allowed })
+    setExternalSourcePermissions(updatedSources)
+    appSettings.setExternalSourcePermissions(updatedSources).catch(reportError)
+  }
+  const getInitialValue = (src: string): boolean =>
+    externalSourcePermissions.find(source => source.type === src)?.allowed ?? false
+
+  return (
+    <Layout>
+      <Caption title={t('headline')} />
```

Comment:
Optional: We probably have to discuss this but in my opinion we should use speaking names for translations as well that tell me what they are instead of where they are used. Reading this I have no idea what kind of text is shown and I think its just an implementation detail.

Reply:
so `title` would be appropriate?